packages/frontend/apps/electron/src/ipc/ipc.module.ts

```
@@ -0,0 +1,123 @@
+    };
+  }
+
+  static forHelper(): DynamicModule {
+    return {
+      module: ElectronIpcModule,
+      imports: [DiscoveryModule],
+      providers: [IpcScanner, ElectronIpcModule],
+      exports: [IpcScanner],
+    };
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)